<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFtpDlForm</class>
 <widget class="QWidget" name="AddFtpDlForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="bt_save">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_cancel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>371</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lbl_qci">
        <property name="text">
         <string>qci</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_filesize">
        <property name="text">
         <string>filesize</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_minThroughput">
        <property name="text">
         <string>minThroughput</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="qciComboFtpDl"/>
      </item>
      <item>
       <widget class="QLineEdit" name="tet_filesize"/>
      </item>
      <item>
       <widget class="QLineEdit" name="tet_minThroughput"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
